plugins/imagetools.md

```diff
@@ -8,7 +8,9 @@ keywords: imagetools rotate rotateleft rotateright flip flipv fliph editimage im
 
 Image Tools (`imagetools`) plugin adds a contextual editing toolbar to the images in the editor. If toolbar is not appearing on image click, it might be that you need to enable `imagetools_cors_hosts` or `imagetools_proxy` (see below).
 
-> *Warning:* This feature requires at least Internet Explorer 10, since it makes use of `HTML5 File API`.
+{{site.svgsNotSupported}}
+
+> **Note**: This feature requires at least Internet Explorer 10, since it makes use of `HTML5 File API`.
```

Comment:
Could we remove this, as we don't support IE 10 or lower anyways?

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1646/commits/0a62b74ab146c8348c11df8d7309ac72e2f8e0ce